### PR TITLE
Add standup guidelines

### PR DESCRIPTION
**Pull request by adiaaida**
### To double check:

* [ ] The right tests are in and and the right validation has happened.  Guidance:  https://github.com/dotnet/core-eng/tree/master/Documentation/Validation

### CODE REVIEW

File: Documentation/TeamProcess/DevGuide/standup.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Standup Guidelines
+
+* Standups should be both physical and Teams meetings
```
Comment (missymessa) [inline Documentation/TeamProcess/DevGuide/standup.md:3]:
```suggestion
* Standups should be both in-person and Teams meetings
```

File: Documentation/TeamProcess/DevGuide/standup.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+* Standups occur twice a week on Tuesdays and Thursdays
+* All team members are expected to attend standup, except when there are conflicts
+* Standups can and will be preemted by meetings like all hands, Q&As, other org/division-wide meetings, and holidays.
```
Comment (missymessa) [inline Documentation/TeamProcess/DevGuide/standup.md:6]:
```suggestion
* Standups can and will be preempted by meetings like all hands, Q&As, other org/division-wide meetings, and holidays.
```

File: Documentation/TeamProcess/DevGuide/standup.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+* The in person standup will take place in the team room, and the person running the meeting should share their screen on the television in the team room.
+* Where possible, an external mic should be used for the standup meeting so that those joining by Teams can hear everyone at the in person meeting.
+* Standup will use the [Standup View](https://github.com/orgs/dotnet/projects/86/views/10) of the .NET Core Engineering Services board.
+  * The board should be organized such that it is grouped by the assignee, with issues in the Backlog (no status) or Done columns not shown.
```
Comment (missymessa) [inline Documentation/TeamProcess/DevGuide/standup.md:10]: nit: these lines have odd indentions.

Comment (adiaaida) [reply Documentation/TeamProcess/DevGuide/standup.md:10]: They're supposed to be sub-bullets of the one above

**Review by missymessa — APPROVED** [review]
LGTM shipit

**Review by riarenas — APPROVED** [review]

**Review by markwilkie — APPROVED** [review]

**Comment by markwilkie** [comment]
Thanks @adiaaida!

**Pull request merged**